cmake/OmimHelpers.cmake

```diff
@@ -33,7 +34,7 @@ function(omim_add_executable executable)
 
   # Enable warnings for all our binaries.
   target_compile_options(${executable} PRIVATE ${OMIM_WARNING_FLAGS})
-  target_include_directories(${executable} PRIVATE ${OMIM_INCLUDE_DIRS})
+  target_include_directories(${executable} SYSTEM PRIVATE ${OMIM_INCLUDE_DIRS})
```

Comment:
Why are they marked as SYSTEM here and in other places? What are the pros and cons of that approach?

Reply:
OMIM_INCLUDE_DIRS contains only boost dir
Marking include directory as SYSTEM helps to remove all warnings from this directory.
GCC workflow doesn't show errors from boost now

Reply:
In that case, I'd prefer to rename it, because strange when OMIM_XXX is SYSTEM.
Name it like 3PARTY_INCLUDE_DIRS